{
  "schema_version": "1.4.0",
  "id": "GHSA-p2p8-49mr-gp6c",
  "modified": "2024-03-25T03:31:43Z",
  "published": "2023-07-06T18:30:51Z",
  "aliases": [
    "CVE-2023-37454"
  ],
  "details": "An issue was discovered in the Linux kernel through 6.4.2. A crafted UDF filesystem image causes a use-after-free write operation in the udf_put_super and udf_close_lvid functions in fs/udf/super.c.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2023-37454"
    },
    {
      "type": "WEB",
      "url": "https://bugzilla.suse.com/show_bug.cgi?id=CVE-2023-37454"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=6f861765464f43a71462d52026fbddfc858239a5"
    },
    {
      "type": "WEB",
      "url": "https://lore.kernel.org/all/00000000000056e02f05dfb6e11a%40google.com/T"
    },
    {
      "type": "WEB",
      "url": "https://lore.kernel.org/all/00000000000056e02f05dfb6e11a@google.com/T"
    },
    {
      "type": "WEB",
      "url": "https://syzkaller.appspot.com/bug?extid=26873a72980f8fa8bc55"
    },
    {
      "type": "WEB",
      "url": "https://syzkaller.appspot.com/bug?extid=60864ed35b1073540d57"
    },
    {
      "type": "WEB",
      "url": "https://syzkaller.appspot.com/bug?extid=61564e5023b7229ec85d"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-416"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2023-07-06T17:15:14Z"
  }
}